packages/icons/src/index.spec.js

```diff
@@ -116,11 +116,11 @@ describe('pcln-icons', () => {
     test.each(iconList)(
       '%s Icon should render with no outline ',
       (key, Component) => {
```

Comment:
if you are using query, the test needs to be async

Reply:
Replaced w/ `getAllByTitle`